lib/cylc/gui/view_tree.py

```diff
@@ -264,8 +264,8 @@ def sort_column(self, model, iter1, iter2, col_num):
         return self._nat_cmp(prop1, prop2)
 
     def default_sort_column(self, model, iter1, iter2):
-        point_string1 = model.get_value(iter1, 0)
-        point_string2 = model.get_value(iter2, 0)
+        point_string1 = model.get_value(iter1, 1)
+        point_string2 = model.get_value(iter2, 1)
```

Comment:
These variables now have bad names (as @kinow has pointed out, the `model.get_value()` is now retrieving the task name, not the cycle point) - could you just change that @GlennCreighton, just to help avoid confusion in future?

It does seem somewhat surprising that sorting on task name intead of cycle point results in correct sorting by cycle point (in the integer case, with single- and double-digit points at the same time) ... but given this GUI is at end-of-life I don't think we need to spend much time worrying about that.